dev/Prototypes/ItemsViewPrototype/FileExplorerScenario.xaml

```diff
@@ -33,19 +55,35 @@
         <l:ViewDefinition Layout="{StaticResource flowLayout}" x:Name="flatFlowDefinition">
             <l:ViewDefinition.ItemTemplate>
                 <c:RecyclingElementFactory RecyclePool="{StaticResource recyclePool}">
-                    <DataTemplate x:Key="GridItemTemplate">
+                    <DataTemplate x:Key="GridItemTemplate" x:DataType="dep:Item">
                         <l:ItemContainer Layout="{StaticResource gridLayout}" Background="LightGray" BorderBrush="Gray" BorderThickness="1">
-                            <TextBlock Text="{Binding Rank}" />
-                            <TextBlock Text="{Binding Mountain}" />
-                            <TextBlock Text="{Binding Height_m}" />
-                            <TextBlock Text="{Binding Range}" />
+                            <TextBlock Text="{x:Bind Rank}" />
+                            <TextBlock Text="{x:Bind Mountain}" />
+                            <TextBlock Text="{x:Bind Height_m}" />
+                            <TextBlock Text="{x:Bind Range}" />
+                        </l:ItemContainer>
+                    </DataTemplate>
+                </c:RecyclingElementFactory>
+            </l:ViewDefinition.ItemTemplate>
+        </l:ViewDefinition>
+
+        <l:ViewDefinition Layout="{StaticResource iconFlowLayout}" x:Name="iconFlowDefinition">
+            <l:ViewDefinition.ItemTemplate>
+                <c:RecyclingElementFactory RecyclePool="{StaticResource recyclePool}">
+                    <DataTemplate x:Key="GridItemTemplate" x:DataType="dep:Item">
```

Comment:
>GridItemTemplate [](start = 41, length = 16)

we are using the same 'recyclePool' instance, so each template should have a unique key. This is used for the template above as well which means these two templates might be used interchangeably, causing weirdness.

Reply:
will fix.